[Bug]: Compile failure on Windows

### Summary

Windows platform builds are riddled with errors

### Affected component

runtime/daemon

### Severity

S2 - degraded behavior

### Current behavior

Exit code is non zero

### Expected behavior

Compile successfully.

### Steps to reproduce

```bash
cargo build --release
```

### Impact

Affected users: Yes
Frequency: always
Consequence: serious

### Logs / stack traces

```text
Compiling zeroclaw v0.1.7 (F:\rust\zeroclaw) 
error[E0428]: the name `default_wasm_fuel_limit` is defined multiple times 
 --> src\config\schema.rs:2863:1 
 | 
 810 | fn default_wasm_fuel_limit() -> u64 { 
 | ----------------------------------- previous definition of the value `default_wasm_fuel_limit` here 
... 
2863 | fn default_wasm_fuel_limit() -> u64 { 
 | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ `default_wasm_fuel_limit` redefined here 
 | 
 = note: `default_wasm_fuel_limit` must be defined only once in the value namespace of this module 
 
error[E0428]: the name `default_wasm_memory_limit_mb` is defined multiple times 
 --> src\config\schema.rs:2867:1 
 | 
 806 | fn default_wasm_memory_limit_mb() -> u64 { 
 | ---------------------------------------- previous definition of the value `default_wasm_memory_limit_mb` here 
... 
2867 | fn default_wasm_memory_limit_mb() -> u64 { 
 | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ `default_wasm_memory_limit_mb` redefined here 
 | 
 = note: `default_wasm_memory_limit_mb` must be defined only once in the value namespace of this module 
 
error[E0433]: failed to resolve: could not find `WasmConfig` in `config` 
 --> src\onboard\wizard.rs:199:30 
 | 
199 | wasm: crate::config::WasmConfig::default(), 
 | ^^^^^^^^^^ could not find `WasmConfig` in `config` 
 | 
help: a struct with a similar name exists 
 | 
199 - wasm: crate::config::WasmConfig::default(), 
199 + wasm: crate::config::CostConfig::default(), 
 | 
help: consider importing this struct 
 | 
 1 + use crate::config::schema::WasmConfig; 
 | 
help: if you import `WasmConfig`, refer to it directly 
 | 
199 - wasm: crate::config::WasmConfig::default(), 
199 + wasm: WasmConfig::default(), 
 | 
 
error[E0433]: failed to resolve: could not find `WasmConfig` in `config` 
 --> src\onboard\wizard.rs:563:30 
 | 
563 | wasm: crate::config::WasmConfig::default(), 
 | ^^^^^^^^^^ could not find `WasmConfig` in `config` 
 | 
help: a struct with a similar name exists 
 | 
563 - wasm: crate::config::WasmConfig::default(), 
563 + wasm: crate::config::CostConfig::default(), 
 | 
help: consider importing this struct 
 | 
 1 + use crate::config::schema::WasmConfig; 
 | 
help: if you import `WasmConfig`, refer to it directly 
 | 
563 - wasm: crate::config::WasmConfig::default(), 
563 + wasm: WasmConfig::default(), 
 | 
 
warning: unused import: `run_tool_call_loop` 
 --> src\gateway\ws.rs:14:74 
 | 
14 | build_shell_policy_instructions, build_tool_instructions_from_specs, run_tool_call_loop, 
 | ^^^^^^^^^^^^^^^^^^ 
 | 
 = note: `#[warn(unused_imports)]` (part of `#[warn(unused)]`) on by default 
 
warning: unexpected `cfg` condition value: `firecrawl` 
 --> src\onboard\wizard.rs:3056:19 
 | 
3056 | #[cfg(feature = "firecrawl")] 
 | ^^^^^^^^^^^^^^^^^^^^^ 
 | 
 = note: expected values for `feature` are: `browser-native`, `channel-lark`, `channel-matrix`, `default`, `fantoccini`, `hardware`, `landlock`, `memory-postgres`, `nusb`, `observability-otel`, `peripheral-rpi`, `probe`, `rag-pdf`, `rppal`, `runtime-wasm`, `sandbox-bubblewrap`, `sandbox-landlock`, `tokio-serial`, `wasm-tools`, and `whatsapp-web` 
 = help: consider adding `firecrawl` as a feature in `Cargo.toml` 
 = note: see <https://doc.rust-lang.org/nightly/rustc/check-cfg/cargo-specifics.html> for more information about checking conditional configuration 
 = note: `#[warn(unexpected_cfgs)]` on by default 
 
warning: unexpected `cfg` condition value: `firecrawl` 
 --> src\onboard\wizard.rs:3075:19 
 | 
3075 | #[cfg(feature = "firecrawl")] 
 | ^^^^^^^^^^^^^^^^^^^^^ 
 | 
 = note: expected values for `feature` are: `browser-native`, `channel-lark`, `channel-matrix`, `default`, `fantoccini`, `hardware`, `landlock`, `memory-postgres`, `nusb`, `observability-otel`, `peripheral-rpi`, `probe`, `rag-pdf`, `rppal`, `runtime-wasm`, `sandbox-bubblewrap`, `sandbox-landlock`, `tokio-serial`, `wasm-tools`, and `whatsapp-web` 
 = help: consider adding `firecrawl` as a feature in `Cargo.toml` 
 = note: see <https://doc.rust-lang.org/nightly/rustc/check-cfg/cargo-specifics.html> for more information about checking conditional configuration 
 
warning: unexpected `cfg` condition value: `firecrawl` 
 --> src\onboard\wizard.rs:3127:19 
 | 
3127 | #[cfg(feature = "firecrawl")] 
 | ^^^^^^^^^^^^^^^^^^^^^ 
 | 
 = note: expected values for `feature` are: `browser-native`, `channel-lark`, `channel-matrix`, `default`, `fantoccini`, `hardware`, `landlock`, `memory-postgres`, `nusb`, `observability-otel`, `peripheral-rpi`, `probe`, `rag-pdf`, `rppal`, `runtime-wasm`, `sandbox-bubblewrap`, `sandbox-landlock`, `tokio-serial`, `wasm-tools`, and `whatsapp-web` 
 = help: consider adding `firecrawl` as a feature in `Cargo.toml` 
 = note: see <https://doc.rust-lang.org/nightly/rustc/check-cfg/cargo-specifics.html> for more information about checking conditional configuration 
 
warning: unexpected `cfg` condition value: `firecrawl` 
 --> src\onboard\wizard.rs:3140:19 
 | 
3140 | #[cfg(feature = "firecrawl")] 
 | ^^^^^^^^^^^^^^^^^^^^^ 
 | 
 = note: expected values for `feature` are: `browser-native`, `channel-lark`, `channel-matrix`, `default`, `fantoccini`, `hardware`, `landlock`, `memory-postgres`, `nusb`, `observability-otel`, `peripheral-rpi`, `probe`, `rag-pdf`, `rppal`, `runtime-wasm`, `sandbox-bubblewrap`, `sandbox-landlock`, `tokio-serial`, `wasm-tools`, and `whatsapp-web` 
 = help: consider adding `firecrawl` as a feature in `Cargo.toml` 
 = note: see <https://doc.rust-lang.org/nightly/rustc/check-cfg/cargo-specifics.html> for more information about checking conditional configuration 
 
warning: unused import: `zip::ZipArchive` 
 --> src\skills\audit.rs:6:5 
 | 
6 | use zip::ZipArchive; 
 | ^^^^^^^^^^^^^^^ 
 
warning: unexpected `cfg` condition value: `firecrawl` 
 --> src\tools\web_fetch.rs:202:11 
 | 
202 | #[cfg(feature = "firecrawl")] 
 | ^^^^^^^^^^^^^^^^^^^^^ 
 | 
 = note: expected values for `feature` are: `browser-native`, `channel-lark`, `channel-matrix`, `default`, `fantoccini`, `hardware`, `landlock`, `memory-postgres`, `nusb`, `observability-otel`, `peripheral-rpi`, `probe`, `rag-pdf`, `rppal`, `runtime-wasm`, `sandbox-bubblewrap`, `sandbox-landlock`, `tokio-serial`, `wasm-tools`, and `whatsapp-web` 
 = help: consider adding `firecrawl` as a feature in `Cargo.toml` 
 = note: see <https://doc.rust-lang.org/nightly/rustc/check-cfg/cargo-specifics.html> for more information about checking conditional configuration 
 
warning: unexpected `cfg` condition value: `firecrawl` 
 --> src\tools\web_fetch.rs:279:15 
 | 
279 | #[cfg(not(feature = "firecrawl"))] 
 | ^^^^^^^^^^^^^^^^^^^^^ 
 | 
 = note: expected values for `feature` are: `browser-native`, `channel-lark`, `channel-matrix`, `default`, `fantoccini`, `hardware`, `landlock`, `memory-postgres`, `nusb`, `observability-otel`, `peripheral-rpi`, `probe`, `rag-pdf`, `rppal`, `runtime-wasm`, `sandbox-bubblewrap`, `sandbox-landlock`, `tokio-serial`, `wasm-tools`, and `whatsapp-web` 
 = help: consider adding `firecrawl` as a feature in `Cargo.toml` 
 = note: see <https://doc.rust-lang.org/nightly/rustc/check-cfg/cargo-specifics.html> for more information about checking conditional configuration 
 
warning: unexpected `cfg` condition value: `web-fetch-html2md` 
 --> src\tools\web_fetch.rs:106:23 
 | 
106 | #[cfg(feature = "web-fetch-html2md")] 
 | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 
 | 
 = note: expected values for `feature` are: `browser-native`, `channel-lark`, `channel-matrix`, `default`, `fantoccini`, `hardware`, `landlock`, `memory-postgres`, `nusb`, `observability-otel`, `peripheral-rpi`, `probe`, `rag-pdf`, `rppal`, `runtime-wasm`, `sandbox-bubblewrap`, `sandbox-landlock`, `tokio-serial`, `wasm-tools`, and `whatsapp-web` 
 = help: consider adding `web-fetch-html2md` as a feature in `Cargo.toml` 
 = note: see <https://doc.rust-lang.org/nightly/rustc/check-cfg/cargo-specifics.html> for more information about checking conditional configuration 
 
warning: unexpected `cfg` condition value: `web-fetch-html2md` 
 --> src\tools\web_fetch.rs:110:27 
 | 
110 | #[cfg(not(feature = "web-fetch-html2md"))] 
 | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 
 | 
 = note: expected values for `feature` are: `browser-native`, `channel-lark`, `channel-matrix`, `default`, `fantoccini`, `hardware`, `landlock`, `memory-postgres`, `nusb`, `observability-otel`, `peripheral-rpi`, `probe`, `rag-pdf`, `rppal`, `runtime-wasm`, `sandbox-bubblewrap`, `sandbox-landlock`, `tokio-serial`, `wasm-tools`, and `whatsapp-web` 
 = help: consider adding `web-fetch-html2md` as a feature in `Cargo.toml` 
 = note: see <https://doc.rust-lang.org/nightly/rustc/check-cfg/cargo-specifics.html> for more information about checking conditional configuration 
 
warning: unexpected `cfg` condition value: `web-fetch-plaintext` 
 --> src\tools\web_fetch.rs:118:23 
 | 
118 | #[cfg(feature = "web-fetch-plaintext")] 
 | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 
 | 
 = note: expected values for `feature` are: `browser-native`, `channel-lark`, `channel-matrix`, `default`, `fantoccini`, `hardware`, `landlock`, `memory-postgres`, `nusb`, `observability-otel`, `peripheral-rpi`, `probe`, `rag-pdf`, `rppal`, `runtime-wasm`, `sandbox-bubblewrap`, `sandbox-landlock`, `tokio-serial`, `wasm-tools`, and `whatsapp-web` 
 = help: consider adding `web-fetch-plaintext` as a feature in `Cargo.toml` 
 = note: see <https://doc.rust-lang.org/nightly/rustc/check-cfg/cargo-specifics.html> for more information about checking conditional configuration 
 
warning: unexpected `cfg` condition value: `web-fetch-plaintext` 
 --> src\tools\web_fetch.rs:122:27 
 | 
122 | #[cfg(not(feature = "web-fetch-plaintext"))] 
 | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 
 | 
 = note: expected values for `feature` are: `browser-native`, `channel-lark`, `channel-matrix`, `default`, `fantoccini`, `hardware`, `landlock`, `memory-postgres`, `nusb`, `observability-otel`, `peripheral-rpi`, `probe`, `rag-pdf`, `rppal`, `runtime-wasm`, `sandbox-bubblewrap`, `sandbox-landlock`, `tokio-serial`, `wasm-tools`, and `whatsapp-web` 
 = help: consider adding `web-fetch-plaintext` as a feature in `Cargo.toml` 
 = note: see <https://doc.rust-lang.org/nightly/rustc/check-cfg/cargo-specifics.html> for more information about checking conditional configuration 
 
warning: unexpected `cfg` condition value: `firecrawl` 
 --> src\tools\web_search_tool.rs:197:11 
 | 
197 | #[cfg(feature = "firecrawl")] 
 | ^^^^^^^^^^^^^^^^^^^^^ 
 | 
 = note: expected values for `feature` are: `browser-native`, `channel-lark`, `channel-matrix`, `default`, `fantoccini`, `hardware`, `landlock`, `memory-postgres`, `nusb`, `observability-otel`, `peripheral-rpi`, `probe`, `rag-pdf`, `rppal`, `runtime-wasm`, `sandbox-bubblewrap`, `sandbox-landlock`, `tokio-serial`, `wasm-tools`, and `whatsapp-web` 
 = help: consider adding `firecrawl` as a feature in `Cargo.toml` 
 = note: see <https://doc.rust-lang.org/nightly/rustc/check-cfg/cargo-specifics.html> for more information about checking conditional configuration 
 
warning: unexpected `cfg` condition value: `firecrawl` 
 --> src\tools\web_search_tool.rs:294:15 
 | 
294 | #[cfg(not(feature = "firecrawl"))] 
 | ^^^^^^^^^^^^^^^^^^^^^ 
 | 
 = note: expected values for `feature` are: `browser-native`, `channel-lark`, `channel-matrix`, `default`, `fantoccini`, `hardware`, `landlock`, `memory-postgres`, `nusb`, `observability-otel`, `peripheral-rpi`, `probe`, `rag-pdf`, `rppal`, `runtime-wasm`, `sandbox-bubblewrap`, `sandbox-landlock`, `tokio-serial`, `wasm-tools`, and `whatsapp-web` 
 = help: consider adding `firecrawl` as a feature in `Cargo.toml` 
 = note: see <https://doc.rust-lang.org/nightly/rustc/check-cfg/cargo-specifics.html> for more information about checking conditional configuration 
 
error[E0061]: this function takes 3 arguments but 2 arguments were supplied 
 --> src\skills\mod.rs:2108:49 
 | 
2108 | let (dest, files_scanned) = install_local_skill_source(&source, &skills_path) 
 | ^^^^^^^^^^^^^^^^^^^^^^^^^^----------------------- argument #3 of type `bool` is missing 
 | 
note: function defined here 
 --> src\skills\mod.rs:822:4 
 | 
 822 | fn install_local_skill_source( 
 | ^^^^^^^^^^^^^^^^^^^^^^^^^^ 
... 
 825 | allow_scripts: bool, 
 | ------------------- 
help: provide the argument 
 | 
2108 | let (dest, files_scanned) = install_local_skill_source(&source, &skills_path, /* bool */) 
 | ++++++++++++ 
 
warning: unused variable: `approval_manager` 
 --> src\gateway\ws.rs:195:9 
 | 
195 | let approval_manager = { 
 | ^^^^^^^^^^^^^^^^ help: if this is intentional, prefix it with an underscore: `_approval_manager` 
 | 
 = note: `#[warn(unused_variables)]` (part of `#[warn(unused)]`) on by default 
 
Some errors have detailed explanations: E0061, E0428, E0433. 
For more information about an error, try `rustc --explain E0061`. 
warning: `zeroclaw` (lib) generated 15 warnings 
error: could not compile `zeroclaw` (lib) due to 5 previous errors; 15 warnings emitted
```

### ZeroClaw version

v0.1.7

### Rust version

1.93.1

### Operating system

Windows 11

### Regression?

Unknown

### Pre-flight checks

- [x] I reproduced this on the latest main branch or latest release.
- [x] I redacted secrets/tokens from logs.
- [x] I removed personal identifiers and replaced identity-specific data with neutral placeholders.

--- a/src/config/mod.rs
+++ b/src/config/mod.rs
@@ -20,8 +20,8 @@ pub use schema::{
     SecretsConfig, SecurityConfig, SecurityRoleConfig, SkillsConfig, SkillsPromptInjectionMode,
     SlackConfig, StorageConfig, StorageProviderConfig, StorageProviderSection, StreamMode,
     SyscallAnomalyConfig, TelegramConfig, TranscriptionConfig, TunnelConfig, UrlAccessConfig,
-    WasmCapabilityEscalationMode, WasmModuleHashPolicy, WasmRuntimeConfig, WasmSecurityConfig,
-    WebFetchConfig, WebSearchConfig, WebhookConfig,
+    WasmCapabilityEscalationMode, WasmConfig, WasmModuleHashPolicy, WasmRuntimeConfig,
+    WasmSecurityConfig, WebFetchConfig, WebSearchConfig, WebhookConfig,
 };
 
 pub fn name_and_presence<T: traits::ChannelConfig>(channel: Option<&T>) -> (&'static str, bool) {

--- a/src/config/schema.rs
+++ b/src/config/schema.rs
@@ -2751,11 +2751,11 @@ pub struct WasmRuntimeConfig {
     pub tools_dir: String,
 
     /// Fuel limit per invocation (instruction budget).
-    #[serde(default = "default_wasm_fuel_limit")]
+    #[serde(default = "default_runtime_wasm_fuel_limit")]
     pub fuel_limit: u64,
 
     /// Memory limit per invocation in MB.
-    #[serde(default = "default_wasm_memory_limit_mb")]
+    #[serde(default = "default_runtime_wasm_memory_limit_mb")]
     pub memory_limit_mb: u64,
 
     /// Maximum `.wasm` module size in MB.
@@ -2860,11 +2860,11 @@ fn default_wasm_tools_dir() -> String {
     "tools/wasm".into()
 }
 
-fn default_wasm_fuel_limit() -> u64 {
+fn default_runtime_wasm_fuel_limit() -> u64 {
     1_000_000
 }
 
-fn default_wasm_memory_limit_mb() -> u64 {
+fn default_runtime_wasm_memory_limit_mb() -> u64 {
     64
 }
 
@@ -2890,8 +2890,8 @@ impl Default for WasmRuntimeConfig {
     fn default() -> Self {
         Self {
             tools_dir: default_wasm_tools_dir(),
-            fuel_limit: default_wasm_fuel_limit(),
-            memory_limit_mb: default_wasm_memory_limit_mb(),
+            fuel_limit: default_runtime_wasm_fuel_limit(),
+            memory_limit_mb: default_runtime_wasm_memory_limit_mb(),
             max_module_size_mb: default_wasm_max_module_size_mb(),
             allow_workspace_read: false,
             allow_workspace_write: false,

--- a/src/skills/mod.rs
+++ b/src/skills/mod.rs
@@ -2105,10 +2105,12 @@ pub fn handle_command(command: crate::SkillCommands, config: &crate::config::Con
                     );
                     println!("  Run 'zeroclaw skill list' to verify the new tools are available.");
                 } else {
-                    let (dest, files_scanned) = install_local_skill_source(&source, &skills_path)
-                        .with_context(|| {
-                        format!("failed to install local skill source: {source}")
-                    })?;
+                    let (dest, files_scanned) = install_local_skill_source(
+                        &source,
+                        &skills_path,
+                        config.skills.allow_scripts,
+                    )
+                    .with_context(|| format!("failed to install local skill source: {source}"))?;
                     println!(
                         "  {} Skill installed and audited: {} ({} files scanned)",
                         console::style("✓").green().bold(),

